backend/auth/acl.go

```diff
@@ -141,12 +141,15 @@ func UpdateACL(input *s3.PutBucketAclInput, acl ACL, iam IAMConfig) error {
 	return nil
 }
 
-func checkIfAccountsExist(accs []string, iam IAMConfig) ([]string, error) {
+func checkIfAccountsExist(accs []string, iam IAMService) ([]string, error) {
 	result := []string{}
 
 	for _, acc := range accs {
-		_, ok := iam.AccessAccounts[acc]
-		if !ok {
+		_, err := iam.GetUserAccount(acc)
```

Comment:
Let me think about this one for a new ticket.  This was in the original code that was refactored too.  The benefit about tossing this to the backend would be to avoid grabbing the lock multiple times and validating the data crc multiple times.  But I want to be careful about premature optimization here since we are probably only ever talking about a handful of accounts here. Also, the more we push to the backend, the more we require each IAM service to implement.